src/Grpc.Net.Client/Internal/GrpcCall.cs

```diff
@@ -265,6 +266,35 @@ public Task<TResponse> GetResponseAsync()
             return null;
         }
 
+        private static StatusCode MapHttpStatusToGrpcCode(HttpStatusCode httpStatusCode)
```

Comment:
Sigh. This matches the mapping documentation and Java and Go. However, the C Core library maps differently. I assume this mapping is fine, but I don't know for sure.